modules/control-utility/src/test/java/org/apache/ignite/util/PerformanceStatisticsCommandTest.java

```diff
@@ -45,6 +47,12 @@ public class PerformanceStatisticsCommandTest extends GridCommandHandlerClusterB
     /** */
     public static final String STATUS = "status";
 
+    /** */
+    public static final String ALL_VIEWS = "--all-system-views";
```

Comment:
Do we actually need to provide options here? Why just collecting all views is not a default behavior?